triage/render.js

```diff
@@ -262,7 +262,13 @@ ${text.slice(0, Math.min(text.length, 1500))}
     const started = tsToString(build.started);
     body += `[${started} ${job}](${url})\n`
   }
-  body += `\n\n/label sig/${owner}`
+  body += `\n\n/kind failing-test`;
+  body += '\n<!-- If this is a flake, please add: /kind flake -->';
+  if (owner) {
+    body += `\n\n/sig ${owner}`;
+  } else {
+    body += '\n\n<!-- Please assign a SIG using: /sig SIG-NAME -->';
```

Comment:
FWIW, I would be way more interested in understanding why `owner` is always undefined.  This used to be parsed out of the `[sig-foo]` tag in the test name